src/components/Chat.tsx

```diff
@@ -142,6 +142,9 @@ const Chat: React.FC<ChatProps> = ({
       return;
     }
     const noteContent = await getFileContent(file);
+    if (debug) {
+      console.log("ALL NOTE CONTENT IS", noteContent)
```

Comment:
Should we only show truncated content from the PDF in case the text is super long? And if you intend to keep this debugging message, no need to make it all caps, perhaps some delimiters can make it more visible.